requirements/docs.txt

```diff
@@ -1,4 +1,4 @@
-# SHA1:5ae67d7c6a316c56c300c1225fc1c964267e85ae
+# SHA1:4a8b0d36a861f57d6f9070ad8ebb9cd077f1f0b4
```

Comment:
I tried to update only the relevant packages. But I ran into a bunch of version conflicts. So I updated everything. I don't see any breaking changes from it.